_source/logzio_collections/_technical-notes/chain-of-trust.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Replacing your Logz.io public certificate
+---
+
+To support encrypted shipping of your logs and metrics,
+shipping methods over SSL/TLS use a public certificate
+that’s authenticated by a trust chain, issued by a certificate authority.
+Lower protocols like TCP need guidance on “who to trust”,
+and that guidance includes a certificate that you install in your environment.
+All this is so your data can be read by Logz.io,
+only after you can be sure we are who we claim to be.
+
+At the end of May 2020, this chain of trust expires,
+and we need to rotate certificates
+so you can keep sending your data without any issues.
+
+To make sure there’s a smooth transition,
+we’ll rotate certificates and establish a new chain of trust
+a few days early. This new chain won’t expire until 2038.
+
+This means you’ll need to update any of your data shipping
+configurations that make use of our certificate chain.
+Configurations need to be updated before May 28
+to avoid disruptions in sending your data to Logz.io.
+
+## What this affects
+
+Many of the shipping methods covered in the Logz.io docs are affected,
```

Comment:
maybe say "some effected but not all"?

Reply:
good point. will be in the next push before merging.